### PR TITLE
Ricos theme

### PR DESCRIPTION
**Pull request by mharsat**

### CODE REVIEW

File: packages/ricos-viewer/web/src/RicosTypes.ts [hunk 1]
```diff
@@ -1,18 +1,21 @@
 interface RichContentProps {
   config?: object;
+  decorators?: Decorator[];
   editorKey?: string;
   helpers?: Helpers;
   initialState?: RicosContent;
+  inlineStyleMappers?: InlineStyleMapper[];
```
Comment (Barackos) [inline packages/ricos-viewer/web/src/RicosTypes.ts:7]: `decorators` and `inlineStyleMappers` RicosViewer's props. You needed them here?

File: packages/ricos-viewer/web/src/themeStrategy/themeStrategy.ts [hunk 2]
```diff
@@ -5,15 +5,13 @@ import { defaultTheme } from './defaults';
...
-    return { theme: { ...defaultTheme, ...theme } };
-  }
+  themeGeneratorFunctions?: ThemeGeneratorFunction[],
+  palette?: Palette | PalettePreset,
+  cssOverride?: CssOverride
+): { theme: CssOverride } {
   jss.setup(preset());
-  const themeGenerator = new ThemeGenerator(isViewer, themeProperties as StringThemeProperties);
+  const themeGenerator = new ThemeGenerator(isViewer, palette, themeGeneratorFunctions);
   const styles = jss.createStyleSheet(themeGenerator.getStylesObject());
```
Comment (Barackos) [inline packages/ricos-viewer/web/src/themeStrategy/themeStrategy.ts:14]: Not a must, but you can prevent themeStrategy for calculating a palette (with ThemeGenerator) if palette isn't given (I wrapped it earlier with an `if (!palette)`).

File: packages/ricos-viewer/web/src/RicosTypes.ts (hunk 1 above)
Comment (mharsat) [reply packages/ricos-viewer/web/src/RicosTypes.ts:7]: They are `RichContentViewer` props not `RicosViewer` props. Adding this helps TS users that pass an RCV child

File: packages/ricos-viewer/web/src/themeStrategy/themeStrategy.ts (hunk 2 above)
Comment (mharsat) [reply packages/ricos-viewer/web/src/themeStrategy/themeStrategy.ts]: done

**Pull request merged**